webapp/src/Utils/Color.ts

```diff
@@ -14,28 +14,43 @@ export const convertHexToRgba = (hex: string, alpha: number = 1) => {
 
 const bluePrimaryColors =
     // [  // Mine
-    //     "rgba(29, 53, 224, 0.4)",
-    //     "rgba(1, 115, 214, 0.4)",
-    //     "rgba(0, 222, 121, 0.4)",
-    //     "rgba(0, 211, 204, 0.4)"
+    //     "rgba( 29,  53, 224, 0.4)",
+    //     "rgba(  1, 115, 214, 0.4)",
+    //     "rgba(  0, 222, 121, 0.4)",
+    //     "rgba(  0, 211, 204, 0.4)"
     // ]
     // [  // Gander's
-    //     "rgba(51, 184, 165, 0.4)",
-    //     "rgba(50, 118, 181, 0.4)",
-    //     "rgb(184, 104, 173, 0.4)",
-    //     "rgba(0, 211, 204, 0.4)"
+    //     "rgba( 51, 184, 165, 0.4)",
+    //     "rgba( 50, 118, 181, 0.4)",
+    //     "rgba(184, 104, 173, 0.4)",
+    //     "rgba(  0, 211, 204, 0.4)"
     // ]
     [  // dtracers's
-        "rgba(5,17,245,0.55)",
-        "rgba(16,142,245,0.55)",
-        "rgba(5,224,204, 0.55)",
-        "rgba(0, 211, 204, 0.55)"
+        "rgba(  5,  17, 245, 0.55)",
+        "rgba( 16, 142, 245, 0.55)",
+        "rgba(  5, 224, 204, 0.55)",
+        "rgba(  0, 211, 204, 0.55)"
```

Comment:
what's going on here? lol

Reply:
some numbers were not indented, some were, it looked ugly as hell